x-pack/plugins/siem/public/containers/hosts/details/host_details.gql_query.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import gql from 'graphql-tag';
+
+export const HostDetailsQuery = gql`
```

Comment:
Thanks for breaking this out into it's own query. I feel this pattern of separate queries + shared `es_adapter` is much cleaner across each page.